utility/src/main/java/org/oppia/util/parser/StringUtils.kt

```diff
@@ -0,0 +1,8 @@
+package org.oppia.util.parser
+
+/** StringUtils to handle input strings. */
+object StringUtils {
+  const val OL_TAG = "ordered"
```

Comment:
Please add documentation for this & later tags to provide some context for what they mean.

Reply:
Done.